crates/rpc/src/cairo/ext_py.rs

```diff
@@ -191,6 +191,11 @@ fn map_tx(tx: BroadcastedTransaction) -> Result<TransactionAndClassHashHint, Cal
                 .contract_class
                 .class_hash()
                 .map_err(|_| CallFailure::Internal("Failed to calculate class hash"))?;
+            use starknet_gateway_types::class_hash::ComputedClassHash;
+            let class_hash = match class_hash {
+                ComputedClassHash::Cairo(c) => ClassHash(c.0),
+                ComputedClassHash::Sierra(s) => ClassHash(s.0),
+            };
```

Comment:
I guess we could match repetition by swapping `ComputedClass::hash() -> &ClassHash` for
`ComputedClass::into_class_hash() -> ClassHash` and `ComputedClass::as_class_hash() -> &ClassHash`

Reply:
In general I think we need a better, simpler abstraction for `ClassHash`, `SierraHash`, `CasmHash`, `CairoHash` (which doesn't exist atm). Or maybe more generally, it might be more correct if we do:
```rust
// Add this type.
struct CairoHash(Felt);

// Redefine this one
enum ClassHash {
    Cairo(CairoHash),
    Sierra(SierraHash),
}
```
but I think that's a problem for another day.